### DISCUSSION TITLE
Suggestion: support code block in admonition summary

### DISCUSSION THREAD

**Discussion by jasper-zanjani** [squidfunk/mkdocs-material#2858] [Ideas]
Do you think it would be possible to support code blocks in admonition summaries? I for one would appreciate being able to use it to hide explanations and output in collapsible admonitions. Just a thought! I continue to appreciate all the hard work you're putting into this amazing project!

**Comment by facelessuser** [comment]
Are you talking inline code blocks or block code blocks? Inline should be possible currently.

**Reply by jasper-zanjani** [reply]
You're write, inline is already possible, but I meant multiline blocks

**Reply by facelessuser** [reply]
You mentioned "summaries" so I am assuming you mean the title/header and you are equating it to detail summaries?

If we are talking multiline blocks in titles, that is not formally supported. The syntax (Markdown extension syntax) for admonitions and such doesn't allow it. You could certainly use raw HTML though. I'm not sure how Material would style it, so you may need additional CSS.

**Reply by squidfunk** [reply]
As @facelessuser pointed out, if you really mean summaries – no, not possible. It would need to be supported by Python Markdown or Python Markdown Extensions. However, you wrote:

> I for one would appreciate being able to use it to hide explanations and output in collapsible admonitions.

That confuses me, because it sounds to me that [Details](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=details#collapsible-blocks) is what you want.
